docs/build.md

```
@@ -24,4 +24,53 @@ Now you can use Godot to import project and edit it, or press `F5` to run projec
+mkdir -p ~/.local/share/godot/export_templates/4.5.stable/
+cp templates/* ~/.local/share/godot/export_templates/4.5.stable/
+```
+In your cloned or extracted Text-Forge project directory, run:
+
+Import Text-Forge project into Godot first
+```bash
+godot --headless --import
+```
+
+Export Text-Forge
+```bash
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)